ephios/core/signup/methods.py

```diff
@@ -768,11 +768,7 @@ def get_configuration_form(self, *args, **kwargs):
 
     def render_configuration_form(self, *args, form=None, **kwargs):
```

Comment:
now do we even need this method? As I understand the django docs a simple `{{ form }}` should suffice?